### PR TITLE
RBN-47: Create login page UI

### PR DESCRIPTION
**Pull request by dunjagojkovic**
### What did you implement?

- [x] Create LoginPage UI
- [x] Exclude Header from LoginPage
- [x] Make it responsive

### How can we verify the changes?

Run the app locally

### More info

- Jira: [RBN-47](https://productdock.atlassian.net/browse/RBN-47) 

- Figma: [Design](https://www.figma.com/design/x4SgW80BE24s9gAFHM3gXS/RBC---Novi-Sad%2C-2024?node-id=3110-26138&node-type=frame&t=iEBMogNE1M2cHXXV-0)

### CODE REVIEW

File: src/pages/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import LoginPagePhoto from '../../assets/LoginPagePhoto.svg'
+import CircleLogo from '../../assets/stamp.svg'
+import './LoginPage.css'
+
+export default function LoginPage() {
+  return (
+    <div className='wrap'>
+      <div className='imgHolder'>
+        <img src={CircleLogo} className='imgLogo' />
+        <img src={LoginPagePhoto} alt='photo' className='img' />
```
Comment (alestojanovic) [inline src/pages/LoginPage/LoginPage.tsx]: I suggest renaming this alt text to something more meaningful like `alt='login-screen-image'`

File: src/pages/LoginPage/LoginPage.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import LoginPagePhoto from '../../assets/LoginPagePhoto.svg'
+import CircleLogo from '../../assets/stamp.svg'
+import './LoginPage.css'
+
+export default function LoginPage() {
+  return (
+    <div className='wrap'>
+      <div className='imgHolder'>
+        <img src={CircleLogo} className='imgLogo' />
```
Comment (alestojanovic) [inline src/pages/LoginPage/LoginPage.tsx]: img elements must have an alt prop, either with meaningful text, or an empty string for decorative images

File: src/pages/LoginPage/LoginPage.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
+.wrap {
```
Comment (alestojanovic) [inline src/pages/LoginPage/LoginPage.css]: Please encapsulate this whole CSS file into .module

File: src/pages/LoginPage/LoginPage.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+.imgLogo {
+  z-index: 2;
+  height: 132px;
+  border-radius: 100%;
+  margin-right: -65px;
+  margin-top: -66px;
+}
+
+.img {
```
Comment (alestojanovic) [inline src/pages/LoginPage/LoginPage.css]: Most of these CSS classes need better naming, especially when using same name as HTML tag here. Good way to organise naming descriptively is to prefix it with function that they support e.g. .loginPageImage, .loginPageLogo, .loginPageContainer. loginPageContent etc

**Review by maxencije — DISMISSED** [review]

**Review by alestojanovic — APPROVED** [review]
Looks good, nicely organised solution for removing Header from LoginPage. 🚀

**Review by isidora-b — APPROVED** [review]

**Pull request merged**